pom.xml

```diff
@@ -63,7 +62,7 @@
     <connection>scm:git:git://github.com/jenkinsci/jira-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/jira-plugin.git</developerConnection>
     <url>https://github.com/jenkinsci/jira-plugin</url>
-    <tag>jira-2.4.1</tag>
+    <tag>HEAD</tag>
```

Comment:
`maven-release-plugin` snafu?

Reply:
Kind of, that was just the release process I was following. We can switch to HEAD though, I don't mind.

Reply:
So, if you *are* using `maven-release-plugin`, then the value of `tag` in regular branch versions should always be `HEAD`; it is only set to a tag name during the tagged commit, then reset during the commit which switches back to a snapshot.